### ISSUE TITLE
Self-Attention Mask

### ISSUE THREAD

**Issue by Azharo** [CyberZHG/keras-bert#131]
I noticed that there doesn't seem to be a self-attention mask as an input layer? Isn't the attention mask needed so that the transformers don't attend to [PAD] tokens?

**Comment by CyberZHG** [comment]
It's added by embedding layers.
